src/framework/core/js/ModelTransformationTransforms.js

```diff
@@ -368,32 +384,40 @@ var fluid = fluid || fluid_2_0_0;
         }
 
         fluid.each(options, function (outPath, key) {
-            // write to output path given in options the value <presentValue> or <missingValue> depending on whether key is found in user input
+            // write to output object the value <presentValue> or <missingValue> depending on whether key is found in user input
             var outVal = (value.indexOf(key) !== -1) ? transformSpec.presentValue : transformSpec.missingValue;
-            fluid.model.transform.setValue(outPath, outVal, transformer);
+            fluid.set(output, outPath, outVal);
```

Comment:
yeah - so this is related to changing it to a standardTransformFunction. Previously, the only way of outputting anything from this transform function was by specifying an "outputPath" - and `fluid.model.transform.setValue(outPath, outVal, transformer);` was the part responsible for writing to the output model.

With the change, we now use the general framework for standardOutputTransformFunction (allowing you to either return or write to outputPath, depending on whether the latter is specified).. So we're creating an `output` object, containing the result of this transformation, and return it to let the framework do the writing to outputPath or passing on to the parent transform function (depending on whether outputPath was specified)